### PR TITLE
RHDEVDOCS-2858 Users are no longer forced to choose between the internal OR external log store

### PR DESCRIPTION
**Pull request by rolfedh**
- For versions 4.5+
- https://issues.redhat.com/browse/RHDEVDOCS-2858
- Direct link to doc preview: https://deploy-preview-30832--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-external.html#cluster-logging-collector-legacy-fluentd_cluster-logging-external
- Direct link to doc preview: https://deploy-preview-30832--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-external.html#cluster-logging-collector-legacy-syslog_cluster-logging-external
- Ready for peer review and merge

### CODE REVIEW

**Comment by rolfedh** [comment]
@anpingli  Please review and approve. Thank you.

**Comment by rolfedh** [comment]
@jcantrill confirmed this change in the following Slack thread: https://coreos.slack.com/archives/GGUR75P60/p1616529505302000?thread_ts=1616529140.301000&cid=GGUR75P60

**Review by anpingli — COMMENTED** [review]
/lgtm

**Review by anpingli — COMMENTED** [review]
/lgtm

File: modules/cluster-logging-collector-legacy-syslog.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [id="cluster-logging-collector-legacy-syslog_{context}"]
 = Forwarding logs using the legacy syslog method
 
-You can use the *syslog* RFC3164 protocol to send logs to destinations outside of your {product-title} cluster instead of the default Elasticsearch log store by creating a configuration file and config map. You are responsible for configuring the external log aggregator to receive the logs from {product-title}.
+You can use the *syslog* RFC3164 protocol to send logs to destinations outside of your {product-title} cluster store by creating a configuration file and config map. You are responsible for configuring the external log aggregator to receive the logs from {product-title}.
```
Comment (bergerhoffer) [inline modules/cluster-logging-collector-legacy-syslog.adoc:8]: This says "cluster store". Should "store" have been deleted as well, like it was in the above file?

**Review by bergerhoffer — COMMENTED** [review]
Just one question if a word was supposed to be deleted.

File: modules/cluster-logging-collector-legacy-syslog.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/cluster-logging-collector-legacy-syslog.adoc:8]: Nice catch - my bad.

**Review by bergerhoffer — COMMENTED** [review]
LGTM now!

**Comment by bergerhoffer** [comment]
These files don't exist in 4.5. @rolfedh Can you check whether these changes don't apply to 4.5, or if they need to be made in a different file?

**Comment by rolfedh** [comment]
@bergerhoffer That's fine. Please ignore 4.5 then.

**Pull request merged**